src/app/page.tsx

```diff
@@ -2,15 +2,13 @@ import Image from "next/image";
 
 export default function Home() {
   return (
-    <main className="bg-[#0A142F]">
+    <main className="sm:grid bg-[#0A142F]">
       <div className="relative min-h-screen">
-        <Image
-          src="/page2.svg"
-          alt="Next.js Logo"
-          layout="fill"
-          objectFit="contain"
-          priority
-        />
+        <div className=" m-8 grid min-h-[1000px] bg-red-700 sm:grid-cols-2">
+          <div className=" m-6 gird min-h-[300px] bg-black"></div>
```

Comment:
tags without content should be void tags and should close inside themselves 

BAD
```
<div></div>
```

GOOD
```
</div>
```